packages/launchpad/src/error/BaseError.vue

```diff
@@ -127,10 +141,29 @@ const { t } = useI18n()
 const props = defineProps<{
   gql: BaseErrorFragment
   retry?: () => void
-  onReadDocs?: () => void
 }>()
 
 const markdownTarget = ref()
 const baseError = computed(() => props.gql)
 const { markdown } = useMarkdown(markdownTarget, computed(() => props.gql.errorMessage), { classes: { code: ['bg-error-200'] } })
+
+const getDocsType = (): string => {
+  const { errorType, errorStack } = baseError.value
+
+  // Full list of errors lives in "packages/errors/src/errors.ts", but cannot be imported to Vue
```

Comment:
Yeah if want this list we would have to move it into a separate file and import it on its own, there's a lot of non-frontend baggage that would run if we try to import from it on the front end. That said I think the code around this is fine.